ts/lerna.json

```diff
@@ -4,7 +4,6 @@
   "packages": [
     "@live-compositor/*",
     "live-compositor",
-    "examples/node-examples",
-    "examples/web-examples/*"
```

Comment:
There is a bug in npm which breaks installing deps for packages which use relative path (`@live-compositor/browser-render`). Once browser-render will be released it won't be a problem